Formula/imagemagick.rb

```diff
@@ -69,6 +71,9 @@ def install
       --enable-static
     ]
 
+    args << (build.without? "fontconfig") ? "--without-fontconfig" : "--with-fontconfig"
+    args << (build.without? "freetype") ? "--without-freetype" : "--with-freetype"
+
     if build.without? "modules"
```

Comment:
That's why I ask you to restore `args << "--with-fontconfig=yes" if build.with? "fontconfig"`.